sql-cli/sql_cli/__main__.py

```diff
@@ -1,13 +1,20 @@
+import logging
 from pathlib import Path
+from typing import Optional
 
 from rich import print as rprint
 from typer import Typer
 
 from sql_cli import __version__
+from sql_cli.connections import validate_connections
 from sql_cli.dag_generator import generate_dag
 
 app = Typer(add_completion=False)
 
+NON_SQL_CLI_LOGGERS = [logging.getLogger(name) for name in logging.root.manager.loggerDict]
+for logger in NON_SQL_CLI_LOGGERS:
+    logger.setLevel(logging.ERROR)
```

Comment:
```suggestion
for logger in logging.root.manager.loggerDict.values():
    logger.setLevel(logging.ERROR)
```
Wondering if this would also work 🤔

Reply:
If not, I would use this so that you don't have to iterate twice of the loggers. :)
```suggestion
for name in logging.root.manager.loggerDict:
    logging.getLogger(name).setLevel(logging.ERROR)
```

Reply:
The former did not work because it does not give a logger instance. The latter works. Thanks, nice suggestion .